releng/build.Jenkinsfile

```diff
@@ -37,7 +37,7 @@ pipeline {
                 -PignoreTestFailures=true \
                 --refresh-dependencies \
                 --continue \
-                clean build signJar createLocalMavenRepo \
+                clean build signJar publish \
```

Comment:
Reason PR build is failing is that PR build does not use Jenkinsfile - either you need to make this same change to https://ci.eclipse.org/lsp4j/job/lsp4j-github-pullrequests/configure - or preferably change https://ci.eclipse.org/lsp4j/job/lsp4j-github-pullrequests to be Jenkinsfile based.

Reply:
of course this will not work because different gradle versions

Reply:
The change to job can be done when this gets merged.